.codespellignore

```diff
@@ -0,0 +1,32 @@
+asssign
+commend
+crate
+crystalize
```

Comment:
Can you add some comments (lines starting with #) in this file to explain why some misspellings have to be allowed (e.g. "CrystalizeMana", "Villify" spell, etc.)?